.github/workflows/release.yaml

```diff
@@ -0,0 +1,58 @@
+name: release
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  create:
+    runs-on: "ubuntu-latest"
+    outputs:
+      upload_url: "${{ steps.release.outputs.upload_url }}"
+    steps:
+      - uses: "actions/create-release@v1"
+        id: "release"
+        env:
+          GITHUB_TOKEN: "${{ github.token }}"
+        with:
+          draft: "true"
+          tag_name: "${{ github.ref }}"
+          release_name: "${{ github.ref }}"
+
+  build-mac:
+    needs: "create"
+    runs-on: "macos-latest"
+    steps:
+      - uses: "actions/checkout@v2"
+      - uses: "actions/setup-go@v2"
+        with:
+          go-version: "^1.17"
+      - name: "build"
+        run: |
+          go build -o darwin-zed /cmd/zed
```

Comment:
`./cmd/zed` or else this will try to build from the root directory

Reply:
whoops, nice catch